ops/log.py

```diff
@@ -26,7 +26,25 @@ def emit(self, record):
         self.model_backend.juju_log(record.levelname, self.format(record))
 
 
-def setup_root_logging(model_backend):
+def setup_root_logging(model_backend, *, debug=False, debug_stream=None):
```

Comment:
Please remove debug_stream, as it's not used in the code. We shouldn't add "dead code" (and if we find it, we should remove it)... this is about reducing complexity.

Thanks!

Reply:
So I took the other route, and use debug_stream is not None rather than a 'debug=True'. It functionally does the same thing, but means the parameter *is* used live, and means we avoid patching something that doesn't really need it.